src/HomePage/components/Search/Search.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import "./SearchStyles.css"
+
+export function Search(props: { onSearchMovie: (arg: string) => void; }) {
```

Comment:
This becomes not very readable. Please separate props type and use the following syntax
```
type SearchProps = {
   onSearchMovie: (input: string) => void;
};

const Search:React.FC<SearchProps> = ({ onSearchMovie }) => {
//...
}
```